starters/blog/content/blog/hi-folks/index.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: New Beginnings
 date: "2015-05-28T22:40:32.169Z"
+description: This is a custom description for SEO and Open Graph purposes, rather than the default generated excerpt. Simply add a description field to the frontmatter.
```

Comment:
Just a small nit. I'd make the sample description more generic since it could be used for other purposes too.

Reply:
```suggestion
description: Far far away, behind the word mountains, far from the countries Vokalia and Consonantia, there live the blind texts. 
```